src/components/Changepersonaldata/Changepersonaldata.css

```diff
@@ -0,0 +1,107 @@
+:root {
+  --first-color: #1a73e8;
+  --input-color: #80868b;
+  --border-color: #dadce0;
+
+  --body-font: "Roboto", sans-serif;
+  --normal-font-size: 1rem;
+  --small-font-size: 0.75rem;
+}
+
+*,
+::before,
+::after {
+  box-sizing: border-box;
+}
+body {
+  margin: 50;
```

Comment:
all the general tag styles (like body, h1 etc.) should be in globalStyles/globalStyles.css

For styling your own components & views - only put styling based on classes & id's - otherwise you & the other developers will be overwriting each other's styling